src/powershell/Public/Get-FinOpsCostExport.ps1

```diff
@@ -0,0 +1,150 @@
+
+    if ($httpResponse.StatusCode -eq 200)
+    {
+      
+        $content = $(ConvertFrom-Json -InputObject $httpResponse.Content -Depth 20).Value
+        Write-Verbose "found $($content.count) export items for the scope $scope"
+
+        # Name parameter received
+        if (-not [System.String]::IsNullOrEmpty($Name))
+        {
+            $content = $content | Where-Object { $_.name -like $Name }
+            Write-Verbose "$($content.count) items left after filtering for Name $Name"
+        }
+        if (-not [System.String]::IsNullOrEmpty($DataSet)) 
+        {
+            $content = $content | Where-Object { $_.properties.definition.type -like $DataSet }
+            Write-Verbose "$($content.count) items left after filtering for DataSet $DataSet"
+        }
+        if (-not [System.String]::IsNullOrEmpty($StorageAccountId))
+        {    
+            $content = $content | Where-Object { $_.properties.deliveryInfo.destination.resourceId.tostring() -eq $StorageAccountId }
+            Write-Verbose "found $($content.count) items after filtering for storageaccountid $StorageAccountId"
+        }
+        if (-not [System.String]::IsNullOrEmpty($StorageContainer))
+        {    
+            $content = $content | Where-Object { $_.properties.deliveryInfo.destination.container.tostring() -like $StorageContainer }
+            Write-Verbose "found $($content.count) items after filtering for StorageContainer $StorageContainer"
+        }
+        $exportdetails = @()
+        $content | ForEach-Object {
+ 
+            $item = [PSCustomObject]@{
+
+                Id                   = $_.id
+                Name                 = $_.name
+                Type                 = $_.type
+                eTag                 = $_.eTag
+                ScheduledStatus      = $_.properties.schedule.status
+                ScheduleRecurrence   = $_.properties.schedule.recurrence
+                RecurrencePeriodFrom = $_.properties.schedule.recurrencePeriod.from
+                RecurrencePeriodTo   = $_.properties.schedule.recurrencePeriod.to
+                NextRuntimeEstimate  = $_.properties.nextRunTimeEstimate
+                Format               = $_.properties.format
+                StorageAccountId     = $_.properties.deliveryInfo.destination.resourceId
+                StorageContainer     = $_.properties.deliveryInfo.destination.container
+                RootFolderPath       = $_.properties.deliveryInfo.destination.rootfolderpath
+                DefinitionType       = $_.properties.definition.type
+                DefinitionTimeFrame  = $_.properties.definition.timeframe
+                DatasetGranularity   = $_.properties.definition.dataset.granularity        
+            }
+            $exportdetails += $item
+            return $exportdetails
+        }    
+    }
+    else
+    {
+        Write-Error "Failed to fetch export data: '$($httpResponse.Content)'"
```

Comment:
Move any external strings to the localized strings file. Look at `Deploy-FinOpsHub` for an example.

Reply:
done